@ecomplus/storefront-template/content/products.json

```
@@ -6,7 +6,12 @@
     },
     {
       "type": "product-block",
-      "enabled": true
+      "enabled": true,
+      "sizeGuide": {
+        "active": false,
+        "image": "/img/uploads/headless.png",
```

```suggestion
        "image": "",
```

acho que você não precisa do `active`, basta checar se existe ou não uma imagem configurada para tabela de medidas, não?

---

Melhor ne, vou alterar aqui